content/microservices/basics.md

```diff
@@ -280,6 +280,36 @@ async onApplicationBootstrap() {
 
 If the connection cannot be created, the `connect()` method will reject with the corresponding error object.
 
+#### Extending the ClientProxy
```

Comment:
This heading might be somewhat too "general". Extending the `ClientProxy` (in the context of this section) is exclusively for customizing the error/incoming response seralisation so maybe we should change it to better reflect what we're going to showcase here(?)

Also, I'd suggest moving this section to the end of the page (or before "handling timeouts" as this one is just an example)

Reply:
Good point. I've renamed it to "Custom Message Serialization"